src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Drivers/ContentElasticIndexSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Localization;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Entities;
+using OrchardCore.Localization;
+using OrchardCore.Mvc.ModelBinding;
+using OrchardCore.Search.Elasticsearch.Core.Models;
+using OrchardCore.Search.Elasticsearch.Models;
+using OrchardCore.Search.Elasticsearch.ViewModels;
+
+namespace OrchardCore.Search.Elasticsearch.Drivers;
+
+internal sealed class ContentElasticIndexSettingsDisplayDriver : DisplayDriver<ElasticIndexSettings>
+{
+    internal readonly IStringLocalizer S;
+
+    public ContentElasticIndexSettingsDisplayDriver(IStringLocalizer<ContentElasticIndexSettingsDisplayDriver> stringLocalizer)
+    {
+        S = stringLocalizer;
+    }
+
+    public override IDisplayResult Edit(ElasticIndexSettings settings, BuildEditorContext context)
+    {
+        if (!string.Equals(ElasticsearchConstants.ContentsIndexSource, settings.Source, StringComparison.OrdinalIgnoreCase))
+        {
+            return null;
+        }
+
+        return Initialize<ContentIndexMetadataViewModel>("ContentIndexMetadata_Edit", model =>
+        {
+            var metadata = settings.As<ContentIndexMetadata>();
```

Comment:
Try to add two Content sources

Reply:
You can't add multiple ContentIndexMetadata and you should not need it in the current setup.